fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -849,6 +849,12 @@ public class Config extends ConfigBase {
     @ConfField
     public static int mysql_service_io_threads_num = 4;
 
+    /**
+     * Enable TCP Keep-Alive for MySQL connections. Useful for long-idled connections behind load balancers.
+     */
+    @ConfField
+    public static boolean mysql_service_nio_enable_keep_alive = false;
```

Comment:
I think `true` is a better default behavior. It won't cause any incompatible issue.

Reply:
I agree, I'll make the change. Thanks for reviewing